package.json

```diff
@@ -25,8 +25,9 @@
     "root": true
   },
   "scripts": {
+    "fix-lint": "eslint --ignore-path=.gitignore . --fix",
```

Comment:
I would not add such script (it adds unnecessary noise) as it it can be very easily achieved via `npx run lint -- --fix` or `npx eslint --fix`

Reply:
fixed